### PR TITLE
Asset landing pages copy changes

### PR DESCRIPTION
**Pull request by sennett**
# [Title](https://clubhouseLink)

<please insert a clubhouse link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>
- item 1
  
## How to test 🧪
  <Please explain how to test your changes>
- step 1 ...
    
## Definition of done ✔️

- [ ] Acceptance criteria for each issue met
- [ ] Unit tests written where needed and passing
- [ ] End-to-end tests for happy path
- [ ] Documentation updated <When applicable>
- [ ] Project builds without errors
- [ ] Non-functional requirements met
- [ ] Code reviewed and functionality tested by the reviewer (locally, Heroku, etc.)
- [ ] Feature verified and accepted by Product Owner
- [ ] Project deployed to production environment

### CODE REVIEW

File: features/asset/AssetView.tsx [hunk 1]
```diff
@@ -96,10 +96,10 @@ export function AssetView({ content }: { content: AssetPageContent }) {
           <Text sx={{ display: 'inline', color: 'text.subtitle' }} variant="paragraph1">
             {t(content.descriptionKey)}
           </Text>
-          <AppLink href={content.link}>
+          <AppLink href={t(content.link)}>
             <WithArrow sx={{ display: 'inline', color: 'link', ml: 2 }}>
               <Text sx={{ display: 'inline', color: 'link' }} variant="paragraph1">
-                {t('learn-more')}
+                {t('learn-more')} about {content.symbol}
```
Comment (paszkowskiDamian) [inline features/asset/AssetView.tsx:102]: Unfortunately we need to add another key to the translations. `about {{token}}` will not make any sense in other languages :(

File: public/locales/en/common.json [hunk 2]
```diff
@@ -842,31 +842,31 @@
...
     "btc": {
-      "description": "ETH is the transactional token that facilitates operations on the Ethereum network, being a form of payment for network participants to execute their requested operations on it.",
-      "link": ""
+      "description": "Bitcoin itself cannot be used directly, but needs to be moved onto the Ethereum Blockchain to use it in DeFi. Both wrapped Bitcoin (wbtc) and renBTC deliver the power of Bitcoin with the flexibility of an ERC20 token. Get the most of your Bitcoin Multiplying your exposure or Borrowing.",
+      "link": "https://wbtc.network/"
     },
     "dai": {
       "description": "ETH is the transactional token that facilitates operations on the Ethereum network, being a form of payment for network participants to execute their requested operations on it.",
-      "link": ""
+      "link": "https://wbtc.network/"
```
Comment (paszkowskiDamian) [inline public/locales/en/common.json:854]: `wbtc.network` for dai? 

Any way the page was removed (in my PR) so I guess we remove keys for DAI

Comment (sennett) [reply public/locales/en/common.json:854]: Ah yes I will remove.

File: features/asset/AssetView.tsx (hunk 1 above)
Comment (sennett) [reply features/asset/AssetView.tsx:77]: Ahh yes.  I will fix.

Comment (sennett) [reply features/asset/AssetView.tsx]: Fixed.

**Review by piekczyk — APPROVED** [review]

**Pull request merged**